app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -162,22 +161,6 @@ $(document).on("ready", function() {
           $("#congressForms-common-fields .form-group").slideUp(400, height_changed);
           $(".form-group", legislatorFieldSet).slideUp(400, height_changed);
           $("input[type=submit]").slideUp(400, height_changed);
-          if ($('[id="$CAPTCHA_SOLUTION"]', legislatorFieldSet).length === 0) {
-            setTimeout(function () {
-              emailsSent++;
-              if (emailsSent === rep_ids.length) {
-                allEmailSent();
-              }
-              $("#congressForms-common-fields .form-group").slideUp(400, height_changed);
-              $(".form-group", legislatorFieldSet).slideUp(400, height_changed);
-              legislatorFieldSet.find("span.info-circle").replaceWith('<span class="info-circle success">Sent <i class="icon-ok-circle"></i></span>');
```

Comment:
Without this the "Submitting \<spinning activity circle\>" indicator never resolves. The ideal place to move this would be into the `onLegislatorSuccess` callback, but I'm guessing there's a reason it was made to update the UI unconditionally on a short timeout here, probably because congressforms can sometimes take quite a while to complete a request (and in any case the message might get queued and so `onLegislatorSuccess` won't get fired).